pom.xml

```diff
@@ -899,6 +899,7 @@
                     <exclude>com.puppycrawl.tools.checkstyle.meta.ModuleDetails*</exclude>
                     <exclude>com.puppycrawl.tools.checkstyle.meta.JavadocMetadataScraper*</exclude>
                     <exclude>com.puppycrawl.tools.checkstyle.meta.XmlMeta*</exclude>
+                    <exclude>com.puppycrawl.tools.checkstyle.meta.MetadataGeneratorUtil*</exclude>
```

Comment:
added as `shouldPrintToConsole` is never false in `MetadataGeneratorUtil`, this was to be used by other projects too, so that flag was added, can be removed if necessary.